### PR TITLE
Added Curve

### PR DESCRIPTION
**Pull request by OfficialMuffin**

### CODE REVIEW

File: _data/sites.json [hunk 1]
```diff
@@ -1597,6 +1597,16 @@
         ]
     },
 
+    {
+        "name": "Curve",
+        "url": "https://www.curve.app/",
+        "difficulty": "hard",
+        "notes": "They cannot delete your account immediately as they are obliged by the Money Laundering Regulations 2017 to keep your data for 5 years after you stop using Curve.",
```
Comment (tupaschoal) [inline _data/sites.json]: I understand it might take some time for the data to be fully deleted, but how do you request deletion?

Comment (OfficialMuffin) [reply _data/sites.json]: I forgot to put that in. Users will have to request it via email. I am currently in the process of contacting them about this to clarify this more clearly. I am awaiting their reply.

Comment (tupaschoal) [reply _data/sites.json]: Ok, thanks for the info, let's keep this on hold until you get an answer then

Comment (tupaschoal) [reply _data/sites.json]: Any luck?

**Comment by tupaschoal** [comment]
Closing due to inactivity, please reopen if applicable.

**Pull request closed without merging**